.tekton/osc-operator-pull-request.yaml

```
@@ -14,7 +14,7 @@ metadata:
       files.all.exists(path, !path.matches('must-gather/.*|.tekton/.*must-gather.*yaml$')) &&
       files.all.exists(path, !path.matches('config/peerpods/podvm/.*|.tekton/.*podvm-builder.*yaml$')) &&
       files.all.exists(path, !path.matches('.tekton/.*fbc-.*|fbc/.*')) &&
-      files.all.exists(path, !path.matches('config/manager/.*'))
+      files.all.exists(path, !path.matches('config/manager/manager.yaml'))
```

What about moving the `!` like below?

```suggestion
      !files.all.exists(path, path.matches('config/manager/.*'))
```

This should provide more predictable outputs. I tested it in [CEL Playground](https://playcel.undistro.io/).

---

It's certainly worth a try :)
Now I'm wondering if I should do the same to the other filters? They *seem* to work, but would that be more reliable if I move the `!` there too?

---

yes, I'd do the same on all of them